zowe/zowe-cli
# feat(plugins): add/remove from schema during `zowe plugins install/uninstall`

Signed-off-by: Trae Yelovich <trae.yelovich@broadcom.com>

**What It Does**

This PR addresses the following:
- BugFix: `zowe plugins install` command no longer rebuilds the schema based on its known profile types
  - Now, the command will only add new types to the schema
- Enhancement: `zowe plugins uninstall` now removes a profile type from the schema if no other client is providing it
  - This command now checks the global `extenders.json` file to see if the profile type is contributed by other sources
  - If the only source is the plug-in, the profile type is removed from the schema during uninstallation

**How to Test**

- Using the latest release of Zowe CLI, uninstall a CLI plug-in that provides a profile type
  - Notice that the profile type was not removed during uninstall
- Then, using the CLI build from this PR:
  - Install the same plug-in providing said profile type
  - Uninstall the plug-in

With these changes, only the schema for that profile type will be removed.

**Review Checklist**
I certify that I have:
